scrappers/newsScrapper.ts

```diff
@@ -0,0 +1,43 @@
+import * as cheerio from "cheerio";
+import { Element as CheerioElement, Cheerio, SelectorType } from "cheerio";
+
+export interface News {
+  id: number
+  title: string,
+  description: string,
+  content?: string,
+  photoUrl: string,
+  date: Date,
+}
+
+export async function scrapNews(): Promise<News[]> {
+  const data = await fetch("https://www.mat.umk.pl/wiadomosci/", { method: "GET" });
+  const $ = cheerio.load(await data.text());
+
+  const news: News[] = [];
+
+  $("section.wiadomosc").each((_, element) => {
```

Comment:
Nie dałoby się tego zrobić "bardziej JavaScriptowo" i użyć funkcji map()?

Reply:
Na początku chciałem to zrobić za pomocą `.map()`, ale miałem problem z TypeScriptem. Spojrzę na to jeszcze

Reply:
Już wleciał commit z `.map()`. Problem był taki, że ten `.map()` jest z frameworka `cheerio` i nie jest taki sam jak ten z prototypu Array.